styles/RadioButtons.module.css

```diff
@@ -1,14 +1,25 @@
 .container {
   display: flex;
-  max-width: 800px;
   flex-wrap: wrap;
   flex-direction: row;
   gap: 20px;
-  font-size: 28px;
+  font-size: 20px;
+  border: none;
+  overflow: scroll;
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
+  scrollbar-width: none; /* Firefox */
 }
+.container::-webkit-scrollbar {
+  display: none; /* Safari and Chrome */
+}
+
+.button,
+.button:focus {
```

Comment:
The only thing I would add is an extra border style for the `:focus` state (when the keyboard focus is on a button, whether that button is selected or not).